doc/setup/installing_macos.adoc

```diff
@@ -0,0 +1,10 @@
+= Installing OSI on MacOS
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+Et malesuada fames ac turpis egestas.
+Mauris pharetra et ultrices neque ornare aenean euismod elementum nisi.
+Nulla facilisi morbi tempus iaculis urna id. Justo eget magna fermentum iaculis.
+Sed augue lacus viverra vitae congue eu. Velit ut tortor pretium viverra suspendisse.
+Amet commodo nulla facilisi nullam vehicula ipsum a.
+Nibh nisl condimentum id venenatis a.
+Diam vel quam elementum pulvinar etiam non quam lacus suspendisse.
```

Comment:
@max-rosin This is actually included in the document.

Reply:
Include commented out.